sig-architecture/api-review-process.md

```diff
@@ -4,8 +4,8 @@
 
 # Process Overview and Motivations
 
-To preserve usability and consistency in Kubernetes APIs, changes and additions require oversight.
-The API review process is intended to maintain logical and functional integrity of the API over time,
+To preserve usability and consistency in Kubernetes core APIs, changes and additions require oversight.
+The API review process is intended to maintain logical and functional integrity of an API over time,
```

Comment:
I actually think "the" is correct - we're trying to oversee the whole API and provide consistency across individual API groups.